### DISCUSSION TITLE
UG DQ 04 — Bottom-Up Analysis and Bias

### DISCUSSION THREAD

**Discussion by johnbeve** [Applied-Ontology-Education/Ontology-and-Intel-Analysis#4] [Undergraduate Discussion Questions]
**GitHub Discussion title:** `UG DQ 04 — Bottom-Up Analysis and Bias`

**Maximum length:** 500 words

Complete the Week 4 readings and lectures before responding.

## Prompt

Bottom-up ontology development often begins with existing data, documents, labels, or analytic practices.

Choose one example of intelligence-relevant source material and answer:

1. What categories or distinctions are already present in the source?
2. Which categories appear to reflect the world, and which may instead reflect the source system, analyst, collection process, or institutional perspective?
3. Identify one possible source of bias.
4. How could importing the source terminology directly into an ontology reproduce that bias?
5. What modeling or documentation step could reduce the problem?

**Comment by drewbonge12** [comment]
An example of intelligence-relevant source material is OSINT, or Open Source Intelligence. This form of intelligence creates relevant information out of public media sources, records, and repositories. Some of the categories that are already present in OSINT are the entities, consisting of people, or organizations themselves that are posting information, events that information might be about, the sources, such as twitter that information is posted on, as well as the reliability of those sources. The categories such as entities posting information, and the events they may be posting about both reflect the real world. The sources themselves would reflect a source system, and the reliability of those sources would reflect upon the analysist.
One source of bias that comes with OSINT intelligence comes from the gamification of available information from sources such as twitter, Instagram, reddit, and youtube. OSINT information that scrapes from the previously listed sources will be inherently biased because of algorithms that are designed to push already popular ideas, or content to the systems users. The gamification aspect of media creates an environment where you lose out on most information that is being posted online with the cost of mainly displaying already largely consumed media. For example, an Instagram post made by an entity with 10,000 followers will receive much more engagement than a post from an entity with 100 followers because the Instagram algorithm will recognize that more people are interacting with one account over another, thus favoring popularity over anything else. 
If you were to import source terminology directly into an ontology by using OSINT sources such as Instagram, you would create an ontology that represents only a small fraction of available information. As a result of gamification, the ontology will mainly consist of information that is 'winning' by receiving more interactions, while posts that may contain incredibly useful information will 'lose', thus creating a bias where you lose out on a lot of information because it is less popular. 
The best way to reduce the gamification of social media sources in an ontology would be for an analyst to check the reliability of the source they are pulling from. Source reliability will reduce the bias of favoring popularity over reliability because an analyst will be able to recognize the credibility of information based on the information that post consists of, rather than the amount of interactions it receives. By removing engagement biases, an ontology will more accurately represent all information rather than keeping high-interaction based content over the lesser interacted with information.

**Comment by LaibahA** [comment]
One example of intelligence-relevant source material is Human Source Intelligence (HUMINT) reports. This mainly includes intelligence gathered through humans as sources, like through interviews, conversations with other people, etc. In the reports there would be distinctions made on their backgrounds, what role that person has, the relevance they have to the information, and other metadata like when, why, and how that information was obtained. These things can help assess the source and informations credibility. 
While some categories reflect the real world (like the role of a source or the event being described), others reflect the source system (like a ranking of how reliable the source might be), or collection process (like details on how the information was obtained). For example, data about whether the information came from a professional or less professional source describes the characteristics of the source rather than the actual information. 
One possible source of bias could be how an analyst ranks how credible different human sources are. An analyst might judge information differently based on the background of the human source who provided it. Characteristics like their expertise, occupation, history of providing information, and relationship to the information can all influence how the analyst evaluates the information from that person. However, taking the characteristics into consideration when evaluating information doesn't necessarily mean the analyst will evaluate the information inaccurately. A professional in the field they are reporting in, with a great history of accurate reports, might actually be giving more credible information than someone who casually learned something from a conversation. Ultimately, the bias would stem from if the analyst's perception of those characteristics influences their evaluation of the information. 
Importing the source terminology directly into an ontology could reproduce that bias by making the characteristics about the source appear to support the credibility of the information itself, even though they don't guarantee that the information is actually accurate. For example, if a human source in a HUMINT report is identified as a professional in the field they're reporting on, the ontology might label that person as "credible", which can lead to the assumption that the information is credible as well. However, this could reproduce the bias I described since the information could be wrong, even if it's from an experienced professional.
One documentation step that could reduce this problem is to document the limitations around labels like "credible" or "reliable". This way, these labels won't be treated as facts, and the context from which the evaluations are derived is given more importance and visibility. This is important since we want to avoid the subjectivity that comes with evaluating HUMINT reports from becoming embedded into the ontology and treated as facts/the truth.

**Comment by adriankoz-jos** [comment]
Perhaps the most well-studied type of source material in intelligence studies is signals intelligence, or SIGINT. SIGINT is comprised of nearly all information derived from human-made electronic communications-related phenomena. As a result of the wide breadth of the subject, there are several sub-classifications of SIGINT, including communications intelligence (COMINT), telemetry intelligence (FISINT), and electronic intelligence (ELINT). 
The majority of SIGINT, especially telemetry and electronic intelligence, seem to reflect the world due to the minimal degree of interpretation involved in their collection. Communications intelligence, on the other hand, is concerned primarily with the collection of deliberate interpersonal communications. The processes of language interpretation and translation in particular carry an elevated risk of injecting user and institution influenced biases. Preconceived notions on the part of individual analysts about the inherent capabilities of adversaries may lead to incorrect or ignored connections concerning the meaning of specific intercepted communications: for example, an analyst who underestimates the engineering prowess of a hostile actor may dismiss intercepted communications around bomb or weapons building as aspirational or idle chatter, potentially leading to a successful terrorist attack down the line. 
An ontology that reproduces analyst-provided translations uncritically remains vulnerable to the same sort of misinterpretation and bias around language that individuals are. To mitigate this, documentation of intercepted communications in their native languages should be adopted. Additionally, having multiple individuals or computers translate source material could help cut down on bias influencing word choice in translation.
